ci.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
-# Please run it from root project directory
+set -xe
```

Comment:
Yes, `-e` will stop the script as soon as any of the command return non-success result code, `-x` adds verbose logging